sdk/RELEASE.md

```diff
@@ -12,8 +24,6 @@
 
```

Comment:
Should we add `fix(sdk): Add SDK support for setting resource limits on older KFP versions (#11839)` here as well?

Reply:
since that was in 2.12.2 I think we can leave it out